database.go

```diff
@@ -10,10 +10,13 @@ var Db *gorm.DB
 
 const PER_PAGE = 10
 
-func InitializeDb() error {
+func InitializeDb(dbName ...string) error {
```

Comment:
Added this argument, so that you can specify a name of the database to be used. Mostly useful for tests.

The old behaviour remains unchanged.